go.mod

```diff
@@ -0,0 +1,70 @@
+module github.com/aws-controllers-k8s/pipes-controller
+
+go 1.17
```

Comment:
Now `1.19` - I'll update the bootstrap controller to match

Reply:
Updated